modules/maxquant/lfq/main.nf

```diff
@@ -0,0 +1,35 @@
+process MAXQUANT_LFQ {
+    tag "$meta.id"
+    label 'process_long'
+    conda (params.enable_conda ? "bioconda::maxquant=2.0.3.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/maxquant:2.0.3.0--py310hdfd78af_1"
+    } else {
+        container "quay.io/biocontainers/maxquant:2.0.3.0--py310hdfd78af_1"
+    }
+
+    input:
+    tuple val(meta), path(fasta), path(paramfile)
+    path raw
+
+    output:
+    tuple val(meta), path("*.txt"), emit: maxquant_txt
+    path "versions.yml"          , emit: version
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    export PATH=/usr/local/lib/dotnet:/usr/local/lib/dotnet/tools:/opt/conda/envs/nf-core-maxquant/bin:/opt/conda/envs/nf-core-maxquant/lib/dotnet/tools:/opt/conda/envs/nf-core-maxquant/lib/dotnet:\$PATH
```

Comment:
Is this line still needed after updating the containers

Reply:
Did that now.

Conda fails strangely as the latest build (number 1) is available in the containers but not via conda. Might be updated at a later point.

Reply:
it should be available: https://anaconda.org/bioconda/maxquant/files... maybe you force it to use that build in the conda string...

> Did that now.

I think you updated the old version of `main.nf`?

Reply:
Sorry, not sure why this happened. Now with both changes.